### ISSUE TITLE
iframe doesn's scroll on IOS device

### ISSUE THREAD

**Issue by xmontana** [sachinchoolur/lightGallery#672]
Dear all,

I'm trying to use this excelent script, but when I open a external site with iframe it works perfect on desktop / mobile android, but in iPhone (Safari iOS 10) I can't scroll the web content  :-(

**Comment by sachinchoolur** [comment]
Hey @xmontana,
For current use, please add the following css rule into your document.
``` css
.lg-has-iframe .lg-video {
   -webkit-overflow-scrolling: touch;
   overflow: auto;
}
```
I'll release new version with this fix ASAP

**Comment by sachinchoolur** [comment]
The issue has been fixed in version 1.6.1
